### PR TITLE
Added react-present-remote to showcase.

### PR DESCRIPTION
**Pull request by limscoder**
Added app to the showcase: https://github.com/limscoder/react-present/tree/master/remote

Website builds and displays properly.

### CODE REVIEW

**Comment by facebook-github-bot-6** [comment]
By analyzing the blame information on this pull request, we identified @brentvatne and @manonthemat to be potential reviewers.

File: website/src/react-native/showcase.js [hunk 1]
```diff
@@ -689,6 +689,13 @@ var apps = [
     author: 'Jonathan Solichin',
   },
   {
+    name: 'react-present remote',
+    icon: 'https://lh3.googleusercontent.com/o17riYlDAyKI6Yu6grtqfZadk0KFqdjWgqonC-p9-1RhhUTDh8Nh1ZFMToHo2hjA9Aw=w300-rw',
+    link: 'https://github.com/limscoder/react-present/tree/master/remote',
+    linkPlayStore: 'https://play.google.com/store/apps/details?id=com.github.limscoder.ReactPresent',
+    author: 'Dave Thompson @limscoder'
```
Comment (brentvatne) [inline website/src/react-native/showcase.js]: trailing comma here as per style in the rest of the file :)

**Comment by facebook-github-bot-6** [comment]
It's been a while since the last commit was reviewed and the labels show this pull request needs review. Based on the blame information for the files in this pull request we identified @brentvatne as a potential reviewer. Could you take a look please or cc someone with more context?

**Comment by lacker** [comment]
Hey, I'm really sorry for the churn, but we are changing up how the showcase works, and just focusing on apps that have some media presence or technical blog posts / other content. So I appreciate the spirit behind this pull request but I think we should close it.

**Pull request closed without merging**